### ISSUE TITLE
Remove "bad" staging environment?

### ISSUE THREAD

**Issue by christiandornetto** [Azure/static-web-apps#133]
Here's more:

When I try to browse the production url, it actually takes me to the pull request staging environment (which doesn't exist as you can see from the screenshot).

![image](https://user-images.githubusercontent.com/38754730/90402701-bcc60900-e097-11ea-9182-57cb21b33e3c.png)

Staging Url: https://wonderful-coast-0c62a0e03-4.westeurope.azurestaticapps.net/

Actual Prod Url: https://wonderful-coast-0c62a0e03.azurestaticapps.net/

_Originally posted by @christiandornetto in https://github.com/Azure/static-web-apps/issues/131#issuecomment-674889488_

**Comment by christiandornetto** [comment]
As per the title. I have a static web app that is deployed on azure. Today I had a pull request submitted which caused the build pipeline failed because of a simple error which I then corrected. 

I then merged the pull request but this did not delete the staging environment previously created and now when I try to navigate to the production I get redirected to the "dead" staging environment which results in a 404.

Is there a way to restore previous deployments or delete environment using a console or similar?
